editor/find_in_files.h

```diff
@@ -177,13 +179,15 @@ class FindInFilesPanel : public Control {
 
 private:
 	void _on_result_found(String fpath, int line_number, int begin, int end, String text);
+	void _on_button_clicked(TreeItem *item, int column, int id, int mouse_button_index);
```

Comment:
```suggestion
	void _on_button_clicked(TreeItem *p_item, int p_column, int p_id, int p_mouse_button_index);
```
Arguments should be prefixed with `p_`